README.md

```diff
@@ -46,20 +46,20 @@ Additional tools that use the libraries created by the reconciliations are also
 - `kafka-clusters`: Manages Kafka clusters via OCM.
 - `ldap-users`: Removes users which are not found in LDAP search.
 - `openshift-clusterrolebindings`: Configures ClusterRolebindings in OpenShift clusters.
-- `openshift-groups`: Manages OpenShift Groups.
+- `openshift-groups`: [Manages OpenShift Groups.](/docs/integreations/openshift_groups.md)
```

Comment:
while I love the word `integreations` as it is indeed, great, this should be `integrations`

Reply:
oops!